services/task/task.go

```diff
@@ -126,3 +126,25 @@ func CreateMigrateTask(doer, u *user_model.User, opts base.MigrateOptions) (*adm
 
 	return task, nil
 }
+
+// RetryMigrateTask retry a migrate task
+func RetryMigrateTask(repoID int64) error {
+	migratingTask, err := admin_model.GetMigratingTask(repoID)
+	if err != nil {
+		log.Error("GetMigratingTask: %v", err)
+		return err
+	}
+	if migratingTask.Status == structs.TaskStatusQueued || migratingTask.Status == structs.TaskStatusRunning {
+		return nil
+	}
+
+	// Reset task status and messages
+	migratingTask.Status = structs.TaskStatusQueued
```

Comment:
@wxiaoguang  When reset the task, would also reset the storage/database? 
Currently only the reset task is working perfectly (at least in my local area).
Should we wait for the relevant issuer feedback before we consider adding reset storage/database?

Reply:
If it has a clear comment (TODO/FIXME) for this case.

Reply:
> If it has a clear comment (TODO/FIXME) for this case.

Thank you for your review